codejam-virtual-keyboard/index.js

```diff
@@ -0,0 +1,71 @@
+const keyLayoutEng = [
+    "`","1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "-","=","Backspace",
+    "Tab","q", "w", "e", "r", "t", "y", "u", "i", "o", "p", "[","]","\\",
+    "CapsLock", "a", "s", "d", "f", "g", "h", "j", "k", "l", ";","'","Enter",
+    "shift", "z", "x", "c", "v", "b", "n", "m", ",", ".", "?","Shift",
+    "Ctrl","Win","Alt","Space","Alt", "◄", "▲","▼", "►", "Ctrl"
+];
+let wrapper = document.createElement('div');
+wrapper.classList.add('wrapper');
+let textarea = document.createElement('textarea');
+let keyboard = document.createElement('div');
+keyboard.classList.add('keyboard');
+let row1 = document.createElement('div');
+let row2 = document.createElement('div');
+let row3 = document.createElement('div');
+let row4 = document.createElement('div');
+let row5 = document.createElement('div');
+row1.classList.add('row1');
+row2.classList.add('row2');
+row3.classList.add('row3');
+row4.classList.add('row4');
+row5.classList.add('row5');
+document.querySelector('body').append(wrapper);
```

Comment:
use document.body instead of document.querySelector('body')

Reply:
Need to append in body in last step after all your manipulation with elements inside wrapper, it is better for performance